components/fieldlabel/stories/fieldlabel.test.js

```diff
@@ -20,7 +16,7 @@ export const FieldLabelGroup = Variants({
 		{
 			testHeading: "Wrapped",
 			withStates: false,
-			label: "Label example with longer text that will wrap to the next line. And with an asterisk that marks it as required.",
+			label: "Label example with longer text that will wrap to the next line. Sometimes there is an asterisk that marks it as required.",
```

Comment:
I didn't see an asterisk, but I wasn't supposed to, right?
![image](https://github.com/user-attachments/assets/e35166f3-c0a7-4a22-9ebe-07712cf1421d)

Do we want to test it with the asterisk? Or with **and** without the asterisk?

Reply:
Correct, we weren't actually testing for the asterisk, but I see how it's confusing now with that text! 🫠 I don't think it's really necessary to test the required state with the wrapping label, but I could be wrong. So just in case, I removed the `withStates: false`. I also added in `ignore: ["Wrapped"]` to the disabled state data, so let me know what you think! 

It should show the wrapped optional, and wrapped required (with the asterisk) now. https://github.com/adobe/spectrum-css/pull/3165/commits/6bbd57fc16450bcbf6588e040751344760288591